core/resources/containermetadataresource/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package containermetadataresource
```

Comment:
I've actually just removed this package for the UUID, meant to do it earlier. Since we have to have the `storage_key` it makes sense to use that as the primary key, at least, I can't see a reason not to do this.